org-docs/OMBUDS.md

```diff
@@ -9,13 +11,31 @@ The NiPreps Ombuds team is guided by the [Code of Ethics](https://www.ombudsasso
 
 Ombuds consultations are confidential; only in the case of imminent risk or serious harm may your identity be disclosed or your concerns discussed with anyone without your explicit permission.
 
+The Ombuds team is not responsible for making decisions or taking actions on behalf of the community, but rather serves as a resource for community members who are seeking assistance in resolving issues or concerns.
+
 ## Requirements for Ombuds
 
 Effective ombuds are active listeners, able to communicate successfully, and remain nonjudgmental, especially at the initial stages of a report when an individual has taken the courageous step of speaking out.
 At subsequent stages, ombuds demonstrate their problem-solving and conflict-resolution skills, driven to understand all positions deeply.
 Although specific training is recognized and encouraged, the *NiPreps Community* needs more resources to provide formal training, and therefore, specific training is optional to serve as Ombuds.
 
-It is also fundamental for ombuds to be neutral. The most obvious proof of neutrality is that ombuds are not simultaneously involved in any capacity within the *NiPreps Community*. As ombuds, they do not participate in the decision-making or provide advisory services to the different organisms and committees of the institution. Formerly active members who are now inactive are welcome to become Ombuds.
+### Neutrality and Independence
+
+It is fundamental for ombudspersons to be neutral. The most obvious proof of neutrality is that ombuds are not simultaneously involved in any capacity within the *NiPreps Community*. As Ombuds, they do not participate in the decision-making or provide advisory services to the different organisms and committees of the institution. Formerly active members who are now inactive are welcome to become Ombuds.
+
+To ensure the Ombuds team's neutrality and independence, the following guidelines will be followed:
+
+* Ombudspersons should not be members of the NiPreps Steering Committee (NSC) or any other NiPreps governance body (e.g., project maintainers, PIs, etc.).
+* Ombudspersons should have no conflict of interest with respect to the community or its members.
+* Ombudspersons should be neutral and impartial in their approach to disputes or conflicts that arise, however are ready to take actions to protect members of the community at imminent risk.
+* Ombudspersons should not engage in any activity that could compromise their neutrality or independence, including serving as a spokesperson for the community or engaging in any fundraising or advocacy activities on behalf of the community.
+* Ombudspersons are invited to participate in the Community assemblies (*NiPreps bi-monthly Roundups*), however they cannot take part of nonsecret votings. Should an Ombudsperson be present at the voting, the voting must be executed secretly.
```

Comment:
Can you elaborate more on why you find both in contradiction?

BTW. Please note that there are very few (if any) "binding" votes taking place in the Roundups. The governance places decision-making in entities incompatible with serving as ombudsperson. In practice, this ruling just helps in "consultation" voting (i.e., nonbinding). The only binding voting I can think of would be the approval of the first version of the governance (where there are no ombuds yet, btw). So it is virtually impossible to have an ombuds taking part in a decision.

Reply:
I have another question there on Line 32... It says:

> they cannot take part of nonsecret votings

Immediately followed by:

> Should an Ombudsperson be present at the voting, the voting must be executed secretly.

Why allow the Ombudsperson to vote secretly or non-secretly, as this may complicate the governance? It might be better to just ask the Ombudsperson to both abstain from all votes and step out of meetings where a vote is taking place.

Reply:
I see it as like judges, who get to vote in the elections despite their job being all about neutrality.

Having to walk out of a meeting or not being allowed to cast a vote when everyone else can is not very inclusive. Also, ombuds' neutrality should prevail over their own opinions, and that is only achievable if the ombudsperson reflects on their own biases. Finally, the incompatibility argument implies causation between a vote in an assembly referendum and the possibility of independently and neutrally acting later on specific cases. Finally, the secrecy of the voting is necessary to ensure members who need to reach out to ombuds do not draw their own conclusions about neutrality because they could see how the ombuds voted in the past.

Finally, it is also essential to consider the nature of the voting (assembly or referendum, everyone votes), with consultive only character (e.g., choose your favorite logo proposal) over very general things.

I'm not particularly against taking this out, but I honestly believe it is a nice thing that ombuds can be there and are not treated differently.